examples/electricity/two_resistors.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python3
+
+from sympy import solve
+from symplyphysics import Symbol, units
+from symplyphysics.laws.electricity import current_is_proportional_to_voltage as ohm_law
+from symplyphysics.laws.electricity.circuits import resistivity_of_parallel_resistors as parallel_resistors
+from symplyphysics.laws.electricity.circuits import resistivity_of_serial_resistors as serial_resistors
+
+# Two resistors are connected across a 12 V battery with internal resistance of 1 Ohm.
+# When they are connected in parallel, the current in the circuit is 4 A.
+# When they are connected in series, the current in the circuit is 1 A.
+# Find the resistance of said resistors.
+
+R1 = Symbol("R1", units.impedance)
+R2 = Symbol("R2", units.impedance)
+R_battery = Symbol("Rb", units.impedance)
+E_battery = Symbol("E", units.voltage)
+I_series = Symbol("Is", units.current)
+I_parallel = Symbol("Ip", units.current)
+
+# Parallel connection
+
+parallel_law = parallel_resistors.law.subs({parallel_resistors.inv_resistances: (1 / R1, 1 / R2)})
```

Comment:
Use conductivity law to find inverted resistance: https://github.com/blackyblack/symplyphysics/blob/master/symplyphysics/definitions/electrical_conductivity_is_inversed_resistance.py
Also, let's use `conductance` everywhere instead of `inv_resistance`. For this reason, the law for parallel resistors should be updated as well.